akka-http-core/src/main/scala/akka/http/scaladsl/settings/ClientConnectionSettings.scala

```diff
@@ -30,6 +30,8 @@ abstract class ClientConnectionSettings private[akka] () extends akka.http.javad
   def websocketRandomFactory: () ⇒ Random
   def socketOptions: immutable.Seq[SocketOption]
   def parserSettings: ParserSettings
+  def exchangeLogging: Boolean
```

Comment:
Corresponding to above, make that a single settings `logUnencryptedNetworkBytes: Option[Int]`.